### PR TITLE
DIV-5373: Respondent landing page for DaRequested

### PR DESCRIPTION
**Pull request by CoffeeOwl97**
# Description
Setting up respondents landing page for when applying for decree absolute.
Includes bug fix for redirection to respondents landing page getting redirected to applicants landing page.

Fixes # (issue)
[DIV-5373](https://tools.hmcts.net/jira/browse/DIV-5373)

## Type of change
- [x] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: helpers/caseOrchestrationHelper.js [hunk 1]
```diff
@@ -54,12 +54,12 @@ const formatSessionForSubmit = req => {
 const validateResponse = (req, response) => {
   const { idam } = req;
 
-  const userIsRespondentAndDivorceNotGranted = (idam.userDetails.email === response.data.respEmailAddress) && !(response.state === 'DivorceGranted');
+  const userIsRespondentAndNotAllowedIntoDA = (idam.userDetails.email === response.data.respEmailAddress) && !(response.state === 'DivorceGranted' || response.state === 'DARequested');
```
Comment (dtaylor7) [inline helpers/caseOrchestrationHelper.js]: break these conditions down into separate variables to make more readable

File: steps/respondent/progress-bar/RespondentProgressBar.content.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
 
     "daRequested": {
       "daRequestedTitle": "Your application for Decree Absolute is being processed",
-      "daRequestedDescription": "This application is subject to checks to ensure there are no outstanding applications or other matters that require completion before the divorce is finalised"
+      "daRequestedDescription": "This application is subject to checks to ensure there are no outstanding applications or other matters that require completion before the divorce is finalised. If there arte no outstanding applications or other matters that require completion, the court will issues your decree absolute, and your divorce will be complete"
```
Comment (dtaylor7) [inline steps/respondent/progress-bar/RespondentProgressBar.content.json]: add `.` to end

Comment (tchow8) [inline steps/respondent/progress-bar/RespondentProgressBar.content.json]: arte? or are?

Comment (tchow8) [reply steps/respondent/progress-bar/RespondentProgressBar.content.json]: and 'the court will issues you', I think it should just be issue?

Comment (CoffeeOwl97) [reply steps/respondent/progress-bar/RespondentProgressBar.content.json]: Good spots 🙂

Comment (CoffeeOwl97) [reply steps/respondent/progress-bar/RespondentProgressBar.content.json]: Good spot.

**Comment by CoffeeOwl97** [comment]
Added a fix to the hmtl validation timeouts courtesy of @dtaylor7 .

File: steps/respondent/progress-bar/sections/DivorceRequested.html [hunk 3]
```diff
@@ -1,4 +1,11 @@
 <div class="da-requested-content">
-    <h3 class="govuk-heading-m">{{ content.daRequested.daRequestedTitle }}</h3>
+    <h3 class="govuk-heading-m">
+
+        {% if userIsWife %}
```
Comment (qzhou-hmcts) [inline steps/respondent/progress-bar/sections/DivorceRequested.html]: Careful here, this can be same sex couple

Comment (CoffeeOwl97) [reply steps/respondent/progress-bar/sections/DivorceRequested.html]: What is the best way to identify what to place on the page then? We have divorceWho which tells us for the respondent, but I dont think we have anything for the petitioner 😕

File: helpers/caseOrchestrationHelper.js [hunk 4]
```diff
@@ -54,16 +54,21 @@ const formatSessionForSubmit = req => {
 const validateResponse = (req, response) => {
   const { idam } = req;
 
-  const userIsRespondentAndDivorceNotGranted = (idam.userDetails.email === response.data.respEmailAddress) && !(response.state === 'DivorceGranted');
+  const stateIsAwaitingDA = response.state === 'AwaitingDecreeAbsolute';
+  // eslint-disable-next-line max-len
```
Comment (dtaylor7) [inline helpers/caseOrchestrationHelper.js]: can remove

**Pull request closed without merging**